applications/app/views/fragments/audioBody.scala.html

```diff
@@ -1,182 +1,75 @@
-                            case _ => {}
+                        @audio.elements.images.map{ img =>
+                            @fragments.imageFigure(img.images)
                         }
 
+                        <figure data-component="main audio new-player"
+                            id="audio-component-container"
+                            data-media-id="@audio.elements.mainAudio.map{ audioElement => @audioElement.properties.id}"
+                            data-source="@audio.downloadUrl.map(Audio.acastUrl(_, false))"
+                            data-itunes-url="@audio.iTunesSubscriptionUrl.getOrElse("#")"
+                            data-download-url="@audio.downloadUrl.getOrElse("#")">
+                        </figure>
+
+
                         <div class="content__main-column__body" data-component="body">
-                            @media match {
-                                case v: Video => {
-                                    <div class="tonal__standfirst">
-                                    @fragments.standfirst(v)
-                                    </div>
-                                }
-                                case _ => {}
-                            }
-
-                            @fragments.contentMeta(media, page)
-
-                            @media match {
-                                case a: Audio if a.fields.body.nonEmpty => {
-                                    <div class="from-content-api">
-                                    @Html(a.fields.body)
-                                    </div>
-                                }
-                                case _ => {}
-                            }
-
-                            @fragments.submeta(media)
-                        </div>
-                    </div>
-                </div>
 
-                @media match {
-                    case v: Video => {
-                        @if(!isPaidContent) {
-                            <div class="content__secondary-column content__secondary-column--media content__secondary-column--video hide-on-childrens-books-site"
-                            aria-hidden="true">
-                                <div class="js-video-components-container"></div>
-                            </div>
-                        }
-                    }
-                    case a: Audio if a.fields.body.nonEmpty => {
-                        <div class="content__secondary-column content__secondary-column--media content__secondary-column--audio hide-on-childrens-books-site"
-                        aria-hidden="true">
-                        @fragments.commercial.standardAd("right", Seq("dark"), Map("desktop" -> Seq("300,250", "300,274", "fluid")))
-                        </div>
-                    }
-                    case _ => {}
-                }
-            </div>
-        </div>
-        </article>
-
-        @if(mediaType != "video" && mediaType != "audio" && isPaidContent) {
```

Comment:
I took this block out because this page will always be audio, so the condition would never be true